src/main/java/com/assignment/StudentInfo.java

```diff
@@ -0,0 +1,98 @@
+package com.assignment;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+
+public class StudentInfo {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+    ArrayList<Student> studentArrayList = new ArrayList<>();
+
+    public void setStudentInfo(String[] info) {
+        Student student = new Student();
+
+        student.setId(info[0]);
+        student.setName(info[1]);
+        student.setAddress(info[2]);
+        student.setStudentClass(info[3]);
+        student.setSchool(info[4]);
+        studentArrayList.add(student);
+        logger.info("Student information added");
+    }
+
+    public ArrayList<Student> getStudentList() {
+        Student student;
+        ArrayList<Student> studentList = new ArrayList<>();
+        if (!studentArrayList.isEmpty()) {
+            for (int i = 0; i < studentArrayList.size(); i++) {
+                student = studentArrayList.get(i);
+                studentList.add(student);
+                logger.info(student.getId() + " " + student.getName() +
+                        " " + student.getAddress() + " " + student.getStudentClass() +
+                        " " + student.getSchool());
+            }
+        } else {
+            logger.info("There is no student information");
+        }
+        return studentList;
```

Comment:
why can't we directly return the `studentArrayList`, do we really need `studentList`?